### DISCUSSION TITLE
xduts使用问题

### DISCUSSION THREAD

**Discussion by peachk1024** [note286/xduts#208] [Q&A] [answered]
![tex](https://github.com/user-attachments/assets/bb52234f-d78f-4127-b813-8e4941377c8d)
我是真一点不会，按照readme操作之后，怎么做呢，在\xduts-main\xduts-main下新建.tex文件吗？还有其他要做的吗？排版、格式、学校logo怎么搞呀，自己敲吗

**Comment by note286** [comment]
请阅读手册2.4节。

**Comment by note286** [comment]
详细说几句吧，建议你安装TeX Live 2024，可以更新所有包，命令为`tlmgr update --all --self`，也可以不更新。然后使用`texdoc xduts`命令打开手册，认真阅读手册即可使用，如果你是研究生，主要阅读第2.4节即可。

**Comment by peachk1024** [comment]
![tex2](https://github.com/user-attachments/assets/7e3731d4-1eb5-42d8-bcf2-8aacf8efe491)
报错The 'cite' package and biblatex are incompatible.

**Reply by peachk1024** [reply]
这个报错我解决了，但是下面三个警告没解决

**Comment by peachk1024** [comment]
![tex3](https://github.com/user-attachments/assets/549cf331-7f51-4022-aa3a-6cfd67632f1c)
还有下面三个警告不知道如何解决

**Reply by peachk1024** [reply]
还有一个警告Package biblatex Warning: Macro 'byeditor+others' already defined. Using \renewbibmacro.，没有解决，其他的都解决了，请教一下这个警告如何解决

**Reply by note286** [reply]
https://github.com/hushidong/biblatex-gb7714-2015/issues/193

**Reply by peachk1024** [reply]
我这个是在线的latex编译环境， https://www.texpage.com/ ，我还是不知道如何解决

**Reply by note286** [reply, marked as answer]
不需要解决。
